editor/selectors.js

```diff
@@ -37,6 +38,20 @@ export function getEditedPostTitle( state ) {
 		: state.editor.edits.title;
 }
 
+export function getEditedPostPreviewLink( state ) {
+	const link = state.currentPost.link;
+	if ( ! link ) {
+		return null;
+	}
+
+	const queryStringPosition = link.indexOf( '?' );
```

Comment:
If we we to add these kind of utilities that could be useful outside the `editor` project, should we create a new root folder `utils` or something like that?